### ISSUE TITLE
labelInnerContent should be a required arg to `labelWithIcon`

### ISSUE THREAD

**Issue by zepumph** [phetsims/scenery-phet#590]
While working on #589, I found it strange and tricky that `KeyboardHelpSection.labelWithIcon` allowed `labelInnerContent` to be optional. I would rather that be required. Note that `labelWithIconList` is already set up this way.  I will fix this and then test.

**Comment by zepumph** [comment]
@jessegreenberg will you please review.

**Comment by jessegreenberg** [comment]
I don't recall why this was ever optional, I agree it should be required. Change looks good, thanks.
